tests/test_main.py

```diff
@@ -156,3 +156,27 @@ def test_indent(tmp_path: Path, indent: int) -> None:
     run(args)
     output = pyproject_toml.read_text()
     assert output == dedent(expected)
+
+
+def test_keep_full_version(tmp_path: Path) -> None:
```

Comment:
test_keep_full_version_cli

Reply:
I will do this, but FWIW I personally consider the full test path (module, class, function names together) when thinking "What does this test?", and I didn't do that because this is already in `test_main`.

Reply:
I disagree personally 😆